src/react-components/object-list.js

```diff
@@ -142,9 +142,17 @@ export default class ObjectList extends Component {
 
   renderExpandedList() {
     return (
-      <div className={styles.presenceList}>
+      <div className={rootStyles.objectList}>
         <div className={styles.contents}>
-          <div className={styles.rows}>{this.state.mediaEntities.map(this.domForEntity.bind(this))}</div>
+          <div className={styles.rows}>
+            {this.state.mediaEntities.length ? (
+              this.state.mediaEntities.map(this.domForEntity.bind(this))
+            ) : (
+              <div className={styles.listItem}>
+                <span>There is no media in the scene yet.</span>
```

Comment:
Use a `<FormattedMessage />` instead with the text in translations.data.json.

Reply:
nit: I would remove the period at the end of the text and center the text.

Reply:
The message is cut off on mobile screens: 
![image](https://user-images.githubusercontent.com/79419/78934892-95638d80-7a60-11ea-867a-804b31655592.png)